packages/server/src/db/db.service.test.ts

```diff
@@ -269,29 +277,39 @@ describe('DBService', () => {
     await new DBService().getShelfInvitationByToken(shelfInvitationToken);
     expect(mockShelfInvitationRepository.findOne).toBeCalledWith(
       { shelfInvitationToken },
-      { relations: ['shelf'] },
+      { relations: ['shelf'] }
     );
   });
 
   it('should save shelf invitation', async () => {
-    mockShelfInvitationRepository.create
-      .mockImplementationOnce(() => mockShelfInvitationEntity);
-    mockShelfRepository.findOne
-      .mockImplementationOnce(() => ({ id: mockShelfInvitationEntity.shelfId }));
+    mockShelfInvitationRepository.create.mockImplementationOnce(
+      () => mockShelfInvitationEntity
+    );
+    mockShelfRepository.findOne.mockImplementationOnce(() => ({
+      id: mockShelfInvitationEntity.shelfId,
+    }));
 
     await new DBService().saveShelfInvitation(mockShelfInvitationEntity);
 
-    expect(mockShelfInvitationRepository.create).toBeCalledWith(mockShelfInvitationEntity);
-    expect(mockShelfRepository.findOne).toBeCalledWith(mockShelfInvitationEntity.shelfId);
+    expect(mockShelfInvitationRepository.create).toBeCalledWith(
+      mockShelfInvitationEntity
+    );
+    expect(mockShelfRepository.findOne).toBeCalledWith(
+      mockShelfInvitationEntity.shelfId
+    );
     expect(mockShelfInvitationRepository.save).toBeCalledWith({
       ...mockShelfInvitationEntity,
       shelf: { id: mockShelfInvitationEntity.shelfId },
     });
   });
 
   it('should delete shelf invitation', async () => {
-    await new DBService().deleteShelfInvitation(mockShelfInvitationEntity.shelfId);
-    expect(mockShelfInvitationRepository.delete).toBeCalledWith(mockShelfInvitationEntity.shelfId);
+    await new DBService().deleteShelfInvitation(
+      mockShelfInvitationEntity.shelfId
+    );
+    expect(mockShelfInvitationRepository.delete).toBeCalledWith(
+      mockShelfInvitationEntity.shelfId
+    );
```

Comment:
I still feel like you have prettier running even though we have it removed. Too many changes that should not be enforced by our current eslint rules make PRs really hard to comprehend.